io-package.json

```diff
@@ -1,8 +1,11 @@
 {
     "common": {
         "name": "cul",
-        "version": "1.1.0",
+        "version": "1.1.1",
         "news": {
+            "1.1.1": {
+                "de": "Senden von FS20 cmdRAW möglich oder über sendTo wie in der Readme beschrieben.",
```

Comment:
JSON is invalid. You can use https://translator.iobroker.in/ to auto translate

Reply:
done